tests/RequestTest.php

```diff
@@ -1033,6 +1033,92 @@ public function testGetRefererInvalid()
         $this->assertNull($request->getReferer());
     }
 
+    /**
+     * @expectedException        Brick\Http\Exception\HttpBadRequestException
+     * @expectedExceptionMessage Invalid protocol: invalid_protocol
+     */
+    public function testGetCurrentShouldReturnHttpBadRequestException()
+    {
+        $request = new Request();
+        $_SERVER['SERVER_PROTOCOL'] = 'invalid_protocol';
+        $request->getCurrent();
+    }
+
+    public function testSetFiles()
+    {
+        $request = new Request();
+        $result = $request->setFiles(['./uploaded_file']);
+        $this->assertInstanceOf(Request::class, $result);
+        $this->assertContains('./uploaded_file', $request->getFiles());
+    }
+
+    public function testSetCookiesShouldRemoveCookieHeader()
```

Comment:
The name does not reflect what the test does: it does not actually check that the header is removed!

Reply:
Ok, I add the ```$this->assertSame('', $result->getHeader('Cookie'));``` to check whether the ```Cookie``` header is removed.

- setCookie to set one cookie value firstly and do assertions.
- setCookie to set empty array and do assertions.